src/main/kotlin/blackjack/view/InputPrompt.kt

```diff
@@ -0,0 +1,7 @@
+package blackjack.view
+
+object InputPrompt {
```

Comment:
> Which would be better: putting the function that handles reprompting with a limited number of attempts inside InputView, or inside the Controller?

`InputView` seems more appropriate to me, since it handles user input. What do you think?